.mailmap

```diff
@@ -1182,6 +1182,7 @@ Pastafarianist <mr.pastafarianist@gmail.com>
 Patrick Lacasse <patrick.m.lacasse@gmail.com>
 Patrick Poitras <acebulf@gmail.com>
 Paul Mandel <paulmandel@google.com>
+Paul Scofield <cerebral.paul@gmail.com> Paul Scofield <Cerebral.Paul@gmail.com>
```

Comment:
I think a second line is needed with
```
Paul Scofield <cerebral.paul@gmail.com> CerebralPaulZ <Cerebral.Paul@gmail.com>
```
I'm not sure where the other name comes from. Might be from GitHub config.